apps/accounts/views.py

```diff
@@ -12,24 +13,37 @@
 from accounts.serializers import UserProfileSerializer
 
 
+def find_user(identity):
+    try:
+        return UserProfile.objects.get(
+            Q(fxa_id=identity['uid']) | Q(email=identity['email']))
+    except UserProfile.DoesNotExist:
+        return None
+    except UserProfile.MultipleObjectsReturned:
+        # This shouldn't happen, so let it raise.
+        raise
```

Comment:
I mostly just wanted to be clear that this was a case that could hypothetically happen. Just so that when it happens someone can look at this and go "oh, Mark thought of this and he was wrong." I should probably add some logging but it will also end up in Sentry.